app/code/Magento/Sitemap/Model/Sitemap.php

```diff
@@ -278,14 +282,6 @@ protected function _getStream()
         }
     }
 
-    /**
-     * Add a sitemap item to the array of sitemap items
-     *
-     * @param DataObject $sitemapItem
-     * @return $this
-     * @deprecated 100.2.0
-     * @see ItemProviderInterface
-     */
```

Comment:
Can you explain why this doc block was removed?

Reply:
Replaced